### DISCUSSION TITLE
Use the "Team Updates" category to share updates with other team members?

### DISCUSSION THREAD

**Discussion by choldgraf** [2i2c-org/team-compass#100] [Ideas]
Hey @2i2c-org/tech-team - in a recent conversation with @GeorgianaElena we thought it might be helpful to have a place to share "team updates" with one another that was asynchronous and that wasn't in one of the public Slack channels for 2i2c. What do people think about using GitHub Discussions for this kind of thing? We could use it to share major updates about availability (e.g., vacation etc) in addition to updates about what kinds of work we're focusing on.

Note that this repository is still public (though likely does not get a lot of 👀)...if we wanted a private space for this we could consider the `meta/` repository Discussions board.

**Comment by damianavila** [comment]
What about having a private channel on Slack? What is the new space on Discussions is bringing to the table compared with a private Slack channel?

**Reply by choldgraf** [reply]
Not much, other than slightly more asynchronous. I'd be totally fine with a private slack too!

**Comment by choldgraf** [comment]
What about, instead of creating a new private channel for team updates, we just make the current `#team-updates` channel private? Most of our development-focused conversation should be doing into the `hub-development` etc channels, and thus far we are mostly using Team Updates to let one another know what we are up to. I think this is OK to have private (available to the 2i2c core team only), and could make people feel more comfortable sharing personal updates etc. What do folks think? (in particular the @2i2c-org/tech-team

**Reply by damianavila** [reply]
I agree with that idea!

**Reply by GeorgianaElena** [reply]
+1 from me too!

**Reply by choldgraf** [reply]
this is now implemented!
